pages/Overview/CeptreTextBased.md

```diff
@@ -2,9 +2,106 @@
 title: Ceptre Text Based
 tags: [getting_started, troubleshooting]
 keywords:
-summary: "In this page we introduce to basic programming constructs for ceptre language and run a small hello world program"
+summary: "In this section we introduce the basic programming constructs of the Ceptre language and 
+run a small "Hello World!" program"
 sidebar: mydoc_sidebar
 permalink: CeptreTextBased.html
 folder: Overview
 ---
 
+## Installing Ceptre
+You can download the Ceptre command line tool for Windows, macOS, or Linux here: 
+https://drive.google.com/drive/folders/0B6BJA78gViuAN3A0WlVkdXBjMk0
+
+Alternatively, you can download the source code and find instructions to build it here: 
```

Comment:
In my last conversation with prof, I remember they told that they don't want to put link to ceptre github repo as it is not in the best visual state. Can we instead just copy paste the instructions from git repo here?

Reply:
If we included the instructions from the repo, we would still need to link it, as it is necessary to download the source files. Since a beginning user is unlikely to want to compile from source anyway, I'll just remove that part.